### PR TITLE
fix #78339 atnews.gr

### LINKED ISSUE #78339: atnews.gr
### Issue URL (Incorrect Blocking)
 [https://atnews.gr/flashboot-3-3/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fatnews.gr%2Fflashboot-3-3%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (2880), <br>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Russian
Other extensions used: | апрпао

### PR DESCRIPTION
**Pull request by stanislav-atr**
#78339

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -117,6 +117,9 @@ smythstoys.com#$#body { overflow: visible!important; }
 ||moneysavingexpert.com/cookie-consent-v*/msmCookieConsent.js
 reifendirekt.de###cookie-diss
 hagebau.de##.js-cookie-hint
+atnews.gr#$#.cli-modal-backdrop { display: none !important; }
+atnews.gr#$##cookie-law-info-bar { display: none !important; }
+atnews.gr#$#body { overflow: visible !important; }
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/cookies_specific.txt:122]: `atnews.gr#@##cookie-law-info-bar` please add to cookie_whitelist

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/cookies_specific.txt:122]: `#cookie-law-info-again` also block this element

File: EnglishFilter/sections/whitelist.txt [hunk 2]
```diff
@@ -37,6 +37,8 @@ eventcairo.com#@#a[href^="https://form.typeform.com/"]
 ! https://github.com/AdguardTeam/SafariConverterLib/issues/4
 ! gamer.no - it came from NorwegianList and it is not normally converted for now
 ||gamer.no/?module=Tumedia\DFProxy\Modules^$badfilter
+! https://github.com/AdguardTeam/AdguardFilters/issues/78339
+@@||atnews.gr^$generichide
```
Comment (Sergey-Lyapin) [inline EnglishFilter/sections/whitelist.txt:41]: why we need this rule?

File: EnglishFilter/sections/foreign.txt [hunk 3]
```diff
@@ -3178,6 +3178,8 @@ autopapa.ge##.top-rek
 gamato3.com##iframe[src^="https://syndication.optimizesrv.com/"]
 ! https://github.com/AdguardTeam/AdguardFilters/issues/77162
 sport24.gr###someplaceholder
+! https://github.com/AdguardTeam/AdguardFilters/issues/78339
+atnews.gr##.top-footer
```
Comment (Sergey-Lyapin) [inline EnglishFilter/sections/foreign.txt:3182]: this block mostly contains they own ads

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/foreign.txt (hunk 3 above)
Comment (stanislav-atr) [reply EnglishFilter/sections/foreign.txt:3182]: Looks like pretty common ads to me, albeit placed on the same domain.

Comment (Sergey-Lyapin) [reply EnglishFilter/sections/foreign.txt:3182]: formally these elements are their own ads (most of elements) and google-news element is 100% not an ad so we should not hide whole `.top-footer` container

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/cookies_specific.txt:122]: #cookie-law-info-again Please remove this element, it appears after accepting cookies.

**Comment by Sergey-Lyapin** [comment]
please hide also these share leftovers
Screenshot 2021-03-29 at 13 46 47

**Pull request merged**